### PR TITLE
Typedef visitor to document parameters

### PR DESCRIPTION
**Pull request by liggitt**
adds a typedef to clarify the parameters of the visitor

updates the unit test to verify each namespace/name pair

### CODE REVIEW

File: pkg/api/persistentvolume/util.go [hunk 1]
```diff
@@ -27,10 +27,13 @@ func getClaimRefNamespace(pv *api.PersistentVolume) string {
 	return ""
 }
 
+// Visitor is called with each object namespace and name, and returns true if visiting should continue
```
Comment (rootfs) [inline pkg/api/persistentvolume/util.go]: object's

File: pkg/api/persistentvolume/util_test.go [hunk 2]
```diff
@@ -31,44 +31,56 @@ func TestPVSecrets(t *testing.T) {
...
+			PersistentVolumeSource: api.PersistentVolumeSource{
+				StorageOS: &api.StorageOSPersistentVolumeSource{
+					SecretRef: &api.ObjectReference{
+						Name:      "Spec.PersistentVolumeSource.StorageOS.SecretRef",
+						Namespace: "storageosns"}}}}},
 	}
 
 	extractedNames := sets.NewString()
-	extractedNamespaces := sets.NewString()
+	extractedNamespacedNames := sets.NewString()
```
Comment (rootfs) [inline pkg/api/persistentvolume/util_test.go]: nit: the name is hard to read

File: pkg/api/persistentvolume/util.go (hunk 1 above)
Comment (liggitt) [inline pkg/api/persistentvolume/util.go]: updated

File: pkg/api/persistentvolume/util_test.go (hunk 2 above)
Comment (liggitt) [inline pkg/api/persistentvolume/util_test.go]: renamed

**Comment by liggitt** [comment]
flaked on https://github.com/kubernetes/kubernetes/issues/49548
/retest

**Comment by smarterclayton** [comment]
/approve

No actual API changes

**Pull request merged**